ops/terraform/modules/stateless/variables.tf

```diff
@@ -33,3 +33,13 @@ variable "is_public" {
   type        = bool
   default     = false
 }
+
+variable "dashboard_name" {
+  description = "Name of the bfd cloudwatch dashboards"
+  type = string
+}
+
+variable "dashboard_namespace" {
+  description = "The namespace in which the dashboards live in"
+  type = string
```

Comment:
Could you be sure to run `terraform fmt` in this module and elsewhere? The alignment's out of order here.